components/Profile.js

```diff
@@ -0,0 +1,165 @@
+  fill: transparent;
+  border-radius: 50%;
+  border: 1px solid var(--section-background);
+`;
+
+const StyledForm = styled.form`
+  background: var(--section-background-contrast);
+  width: 80vw;
+  padding: 1rem;
+  margin-bottom: 1rem;
+  display: flex;
+  flex-direction: column;
+`;
+const StyledDiv = styled.div`
+  padding: 0.2rem;
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+`;
+
+const StyledInput = styled.input`
+  display: inline-block;
+`;
+
+const StyledLabel = styled.label`
+  color: var(--contrast-text);
+`;
+
+const StyledSubmit = styled.button`
+  margin-top: 0.5rem;
+  width: 100%;
+  background: var(--text-on-bright);
+  color: var(--text-on-dark);
+  border-style: none;
+  border-radius: 6px;
+`;
+
+export default function Profile({
+  userNameInitials,
+  userName,
+  userEmail,
+  handleEditTheme,
+  currentUser,
+}) {
+  // const { data: session } = useSession();
+  const [showThemes, setShowThemes] = useState(false);
+
+  const colorSchemes = [
+    {
+      title: "warm",
+      name: "warmTheme",
+      background: "conic-gradient(#fbe050, #fbbd50, #f673a4, #762e72 50%)",
+      text: "#762e72",
+    },
+    {
+      title: "cold",
+      name: "coldTheme",
+      background: "conic-gradient(#50cfe2, #9996fa, #0F1555 50%)",
+      text: "#0F1555",
+    },
+    {
+      title: "neutral",
+      name: "neutralTheme",
+      background: "conic-gradient(white 50%, black 50%)",
+      text: "black",
+    },
+    {
+      title: "high contrast",
+      name: "highContrastTheme",
+      background: "conic-gradient(yellow 50%, black 50%)",
+      text: "black",
+    },
+  ];
+
+  function handleShowThemes() {
+    setShowThemes(!showThemes);
+  }
+
+  return (
+    <>
+      <StyledTitleWrapper>
+        <StyledTitle>Hi {userName}!</StyledTitle>
+        <StyledProfileCircle> {userNameInitials} </StyledProfileCircle>
+        <StyledParagraph>this is your profile-page</StyledParagraph>
+      </StyledTitleWrapper>
+      <StyledParagraph>Account-name: {userEmail}</StyledParagraph>
+      <StyledParagraph>
+        Make yourself at home and turn on your favorite illumination
+      </StyledParagraph>
+
+      <StyledColorSchemesButton onClick={handleShowThemes}>
+        Choose your preferred theme{" "}
+        <Icon path={mdiMagicStaff} size={0.9} color="var(--main-dark)" />
+      </StyledColorSchemesButton>
+      {showThemes && (
+        <StyledForm onSubmit={handleEditTheme}>
+          {colorSchemes.map(({ name, title, background, text }) => (
+            <StyledDiv key={title}>
+              <StyledCircle $background={background} $text={text} />
+              <StyledInput
+                defaultChecked={currentUser.theme === name ? true : false}
+                type="radio"
+                id={name}
+                name="theme"
+                value={name}
+                $background={background}
+              />
+              <StyledLabel htmlFor={name}>{title}</StyledLabel>
```

Comment:
I cannot find any reference input for the label?

Reply:
the reference input is right above the label